### DISCUSSION TITLE
Timestamps with long files are overlapping

### DISCUSSION THREAD

**Discussion by Yorams** [katspaugh/wavesurfer.js#3741] [Q&A]
Timestamps with the timeline plugin with a file of about 5 hours are overlapping. Is it possible to adjust the "defaultTimeInterval" function in plugins/timeline.ts, so that with long files the interval is 6000 or something the like?

![image](https://github.com/katspaugh/wavesurfer.js/assets/13483484/b38e4d0b-a516-4e10-87b0-92c1c1991eb2)

**Comment by jagthedrummer** [comment]
@Yorams, you can completely control that by passing options to the timeline plugin when you instantiate it. https://wavesurfer.xyz/docs/types/plugins_timeline.TimelinePluginOptions

For instance:

```javascript
TimelinePlugin.create({
  timeInterval: 15,
  primaryLabelInterval: 60,
});
```
